templates/_components/registration_form.html

```diff
@@ -0,0 +1,54 @@
+<div class="flex min-h-full flex-col justify-center py-20 sm:px-6 lg:px-8">
+  <div class="sm:mx-auto text-center">
+    <h1 class="text-2xl/9 font-bold tracking-tight text-slate-900">
+      {{ title }}
+    </h1>
+    {% if subtitle %}
+      <p class="mt-2 text-sm text-slate-600">{{ subtitle }}</p>
+    {% endif %}
+  </div>
+
+  <div class="mt-10 space-y-6 sm:mx-auto sm:w-full sm:max-w-[480px]">
+
+    {% if form.non_field_errors %}
+      <div class="rounded-md bg-red-50 p-4">
+        <div class="flex">
+          <div class="shrink-0">
+            {% comment %} Hero Icons: exclamation-circle (mini) icon {% endcomment %}
+            <svg class="size-5 text-red-400" viewBox="0 0 20 20" fill="currentColor" aria-hidden="true" data-slot="icon">
+              <path fill-rule="evenodd" d="M10 18a8 8 0 1 0 0-16 8 8 0 0 0 0 16ZM8.28 7.22a.75.75 0 0 0-1.06 1.06L8.94 10l-1.72 1.72a.75.75 0 1 0 1.06 1.06L10 11.06l1.72 1.72a.75.75 0 1 0 1.06-1.06L11.06 10l1.72-1.72a.75.75 0 0 0-1.06-1.06L10 8.94 8.28 7.22Z" clip-rule="evenodd" />
+            </svg>
```

Comment:
I'm guessing (from the comment) that this is the hero icon?

Reply:
Yes, it's the same one used in the Login form. Next PR will make "components" from the icons to tidy this up.